### DISCUSSION TITLE
stuck

### DISCUSSION THREAD

**Discussion by SyauqiR** [dewanakl/undangan#24] [General]
bg kok stuck di halaman login U ya??

**Comment by dewanakl** [comment]
Boleh sertakan ss sewaktu inspect element mas?

**Reply by dewanakl** [reply]
Oiya, untuk comment pada commit, saya hapus
Di discussion saja, biar yang lain jika ada error yang serupa jadi terbantu.

**Comment by SyauqiR** [comment]
![Screenshot (4)](https://github.com/dewanakl/undangan/assets/75843361/21250c5e-9691-48ff-aa51-6bf2643fcc88)

**Reply by SyauqiR** [reply]
stuck di login gitu mas

**Reply by SyauqiR** [reply]
tapi scripnya belum ada di otak atik mass,mohon bantuannya:)

**Comment by dewanakl** [comment]
Untuk ini, masnya perlu server untuk di localhost-nya karena script js ini bertipe module dan perlu domain yang valid
Jadi masnya bisa menggunakan ekstension live server pada vscode ini.

Thank you

**Comment by SyauqiR** [comment]
mau nanya satu bg,kok pas di up ke hosting stuck di 14/15 ya bg?



Pada Jum, 24 Mei 2024 10.50, Dewana Kretarta Lokeswara <
***@***.***> menulis:

**Comment by SyauqiR** [comment]
Pada Jum, 24 Mei 2024 14.42, Syauqi Ridwan ***@***.***>
menulis:

**Comment by SyauqiR** [comment]
tapi kalo dari vcs dia bisa bg,tapi klo udah di up di hosting stuck di
14/15,kalo di vcs cuman 14/14,kenapa itu bg

Pada Jum, 24 Mei 2024 14.42, Syauqi Ridwan ***@***.***>
menulis:

**Comment by dewanakl** [comment]
Boleh sertakan ss sewaktu inspect element mas

**Comment by SyauqiR** [comment]
![Screenshot (6)](https://github.com/dewanakl/undangan/assets/75843361/baa9dbae-f63f-4da9-85cd-83fc6ac3b6d2)
itu past di hosting mas stuck di 14/15 ,tapi kalo di pake visual studio code bisa

**Reply by dewanakl** [reply]
Boleh ss inspect element mas, agar jadi mengetahui pesan errornya
Sama kalau boleh saran, bisa memakai vercel atau netlify agar tidak ada watermark di pojok bawah

**Reply by SyauqiR** [reply]
![Screenshot (8)](https://github.com/dewanakl/undangan/assets/75843361/f940dcf7-6e5e-4546-938b-23838ad8c734)
![Uploading Screenshot (8).png…]()

**Reply by SyauqiR** [reply]
itu bg

**Reply by dewanakl** [reply]
Coba ss dibagian console mas

**Reply by SyauqiR** [reply]
![Screenshot (9)](https://github.com/dewanakl/undangan/assets/75843361/9a2141d7-28fb-465c-8306-4f99bd4abb88)

**Reply by dewanakl** [reply]
Sepertinya dari 000webhost mas

Coba menggunakan netlify saja mas

**Reply by SyauqiR** [reply]
kalo vercel gmn mass?

**Reply by dewanakl** [reply]
Boleh saja mas

**Reply by SyauqiR** [reply]
![Screenshot (10)](https://github.com/dewanakl/undangan/assets/75843361/3a3866b6-a8f5-4aa5-8e2f-2997dc46b466)

**Reply by SyauqiR** [reply]
jadi kek gitu mass

**Reply by dewanakl** [reply]
Pastikan bahwa folder css, js dll sudah ikut ke upload mas

**Reply by SyauqiR** [reply]
oke mass,ternyata foldernya ketinggalan :(,makasi ya mas udah bimbingin (:

**Reply by dewanakl** [reply]
Okey mas, sama sama

**Comment by SyauqiR** [comment]
mas mau nanya pesannya kok ada yg ngisi Ya padahal linkny belum di share?

Pada Jum, 24 Mei 2024 16.29, Dewana Kretarta Lokeswara <
***@***.***> menulis:

**Reply by dewanakl** [reply]
Bisa deploy untuk backend nya sendiri, untuk step by step ada di readme
